.eslintrc.cjs

```diff
@@ -1,48 +1,12 @@
 module.exports = {
```

Comment:
I am seeing a number of linting errors after creating a new project from this branch.
![image](https://user-images.githubusercontent.com/21334130/148916354-f8eda1ea-c166-452c-9f09-0156135b86b1.png)

Reply:
Wonder how this is possible, the configs are basically identical between the old config and the muban/eslint package. I guess this was already an issue (for some) before my changes, because someone else already reported the last eslint error 2 days ago :)

Anyhow;
- The issues with the mock files requires me to change the tsconfig.json (and add the mock pattern). But this file doesn't exist. Should I copy it over from the base skeleton and make the required changes?
- muban/storybook dep should be fixed with the other change
- changed the `_` to a named parameter
- fixed the regex in `_main.ts`